manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Arturia.ArturiaSoftwareCenter
+PackageVersion: 2.4.5.760

```

```suggestion
PackageVersion: 2.4.5
```
Must match the version shown in Control Panel / Apps&Features